Tests/MeiliSearchIntegrationTests/DocumentsTests.swift

```diff
@@ -1,4 +1,5 @@
 @testable import MeiliSearch
+@testable import MeiliSearchCore
```

Comment:
question: do we require @testable for the core target? 99% of models within are public (applicable to all tests)